modules/oe_translation_epoetry/modules/oe_translation_epoetry_mock/src/Form/MockSettings.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_translation_epoetry_mock\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Mock settings to configure ePoetry for testing with the mock.
+ *
+ * We use this to test on remote environments where we cannot control the
+ * environment variables.
+ */
+class MockSettings extends ConfigFormBase {
```

Comment:
I created this configuration so we can configure paths and authentication for using the mock on the demo site and stuff.